zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/InterpreterPropertyBuilder.java

```diff
@@ -31,6 +31,16 @@ public InterpreterPropertyBuilder add(String name, String defaultValue, String d
     return this;
   }
 
+  public InterpreterPropertyBuilder add(String name,
+                                        String envName,
```

Comment:
OK. Does the code has 100 character per-line limit ? If exceed which format is expected ? thanks~